### DISCUSSION TITLE
Are dependency insights accurate?

### DISCUSSION THREAD

**Discussion by rmclaughlin-nelnet** [community/community#23502] [New to GitHub]
<p>We have been using dependency insights to track vulnerabilitys across all our repos, but lately I have noticed that some vulnerabilities listed, and it says there are &gt; 0 dependents, when I click on the dependents it shows ’ No dependents found’. Is this accurate? If so why is it showing in my list? Does it take a while for these resolved dependents to stop being reported? If so how long?<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/8/8801d2fc1fd43c540460babaf2cf92ebf1a6afe5.png" data-download-href="/uploads/short-url/jpaQlcsu5E6beSPQDYlt2SfEK57.png?dl=1" title="Screen Shot 2022-05-16 at 9.15.30 AM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59625617/181077448-779f8fd6-475c-4b43-9ad8-b3e5602a34ca.png" alt="Screen Shot 2022-05-16 at 9.15.30 AM" data-base62-sha1="jpaQlcsu5E6beSPQDYlt2SfEK57" width="690" height="81" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/8/8801d2fc1fd43c540460babaf2cf92ebf1a6afe5_2_690x81.png, https://aws1.discourse-cdn.com/github/original/3X/8/8/8801d2fc1fd43c540460babaf2cf92ebf1a6afe5.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/8/8801d2fc1fd43c540460babaf2cf92ebf1a6afe5.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/8/8801d2fc1fd43c540460babaf2cf92ebf1a6afe5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screen Shot 2022-05-16 at 9.15.30 AM</span><span class="informations">1027×121 9.25 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/c/dc1f4a2d2d1bcfee2d3eb5d8f989164bb2499975.png" data-download-href="/uploads/short-url/vpi53dF0JHdWi5p0UU05zG6iqhv.png?dl=1" title="Screen Shot 2022-05-16 at 9.15.56 AM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59625617/181077449-dd86e3a7-a99b-4231-9577-d12628143aa5.png" alt="Screen Shot 2022-05-16 at 9.15.56 AM" data-base62-sha1="vpi53dF0JHdWi5p0UU05zG6iqhv" width="690" height="436" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dc1f4a2d2d1bcfee2d3eb5d8f989164bb2499975_2_690x436.png, https://aws1.discourse-cdn.com/github/original/3X/d/c/dc1f4a2d2d1bcfee2d3eb5d8f989164bb2499975.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/d/c/dc1f4a2d2d1bcfee2d3eb5d8f989164bb2499975.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dc1f4a2d2d1bcfee2d3eb5d8f989164bb2499975_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screen Shot 2022-05-16 at 9.15.56 AM</span><span class="informations">970×613 60.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hey there <a class="mention" href="/rmclaughlin-nelnet">@rmclaughlin-nelnet</a> :wave:t3:</p>
<p>So there is an open Issue with our internal team investigating this inaccuracy. In the meantime as we work on a fix, the counts would need to be rebuilt manually. A GitHub employee would have to do this so if you are in need of this, please open a ticket with <a href="https://support.github.com/contact">Support</a>.</p>

**Comment by pacroy** [comment]
Hi @queenofcorgis , is there any update regarding this issue (Dependency Insights does not show accurate data)?
My organization wants to use this feature to see how much our internal Actions (and also reusable workflows) are used. But the report seem not get updated correctly..

**Comment by MichielOda** [comment]
Four years later, and the issue is still occurring...

It doesn't seem to be consistent. Some dependency versions work, while others don't.
For context, I'm an organization admin and Enterprise owner, so this doesn't appear to be an access or permissions issue.
From what I can tell, this only affects GitHub Actions.

<img width="965" height="595" alt="image" src="https://github.com/user-attachments/assets/0fbfe174-64c6-4d45-8180-b781e06bb7f5" />
